### ISSUE TITLE
fix all the broken padding, margin and spaces in the home page

### ISSUE THREAD

**Issue by Saifu0** [Saifu0/promoteanybrand#25]
There are some CSS work needs to be done in the home pages but keep in mind it works for mobile view-port too.
e.g 
- `I'm brand` and `I'm Influencer` button are separated by too many spaces
-  No margin at left and right on services card
- Phone Icon, WhatsApp icon and phone number have unnecessary spaces
And any other CSS fixes which might make UI looks better would be appreciated.

This issue can be divided into more than 1 PR if it is too much of work :P

**Comment by Pranjal7852** [comment]
So far i have achieved this mush with inline styling it is very complicated..
![image](https://user-images.githubusercontent.com/68412756/138325723-81b75f69-5c8d-49e8-bb85-d6f89905ee54.png)

**Comment by Saifu0** [comment]
@Pranjal7852 Yes, I checked it out. Your PR fixes first issue. You can continue working on 2nd and 3rd. I already merged the previous PR, feel free to raise new one.
